.github/workflows/vagrant_up.yaml

```diff
@@ -11,16 +11,35 @@ permissions:
 jobs:
   vagrant-up:
     runs-on: macos-latest
-
     steps:
       - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Get latest version
+        id: get-version
+        run: |
+          TAG=$(git describe --tags --abbrev=0)
+          if ! [[ $TAG =~ ^v[0-9]{2}\.[0-9]{2}\.[0-9]{2}$ ]]; then
+            echo "Invalid tag name: '$TAG'"
+            exit 1
+          fi
+          echo "version=${TAG:1}" >> $GITHUB_OUTPUT
+
       # Installing this is skipped in CI as using it was causing some corruption in
       # attempting to mount the shared folder. The guest and host guest additions
       # versions are close enough that we are fine without it.
       # - run: vagrant plugin install vagrant-vbguest
-      - run: NO_SUBMISSIONS=1 vagrant up ubuntu-22.04
+      - name: Vagrant Up
+        run: NO_SUBMISSIONS=1 vagrant up
```

Comment:
Is there enough disk space to make submissions?  Most developers who are downloading this VM are going to want the sample courses to be set up already.